### PR TITLE
Add Senior Reliability Engineer role

### PR DESCRIPTION
**Pull request by lukemorton**
We introduce progression for Reliability Engineers by combining Senior and RE role expectations.

Todo:

 - [x] Decide if there are any RE specific senior requirements
 - [x] Update job description to make clear differentiation between RE and Senior RE

### CODE REVIEW

File: roles/senior_reliability_engineer.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+# Senior Reliability Engineer
+
+Our Senior Reliability Engineers are champions of the Made Tech reliability engineering culture and devops in it's truest and most collaborative form. Through coaching and mentoring our reliability engineering team as well as remaining hands-on, our Senior Reliability Engineers ensure the quality of our customers applications through their lifetime.
```
Comment (chrisblackburn) [inline roles/senior_reliability_engineer.md]: its instead of it's
DevOps instead of devops

File: roles/senior_reliability_engineer.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+## What does the job entail?
+
+We primarily write and deliver custom software to our customers. We build solutions in a variety of languages and platforms, historically Ruby on Rails, we tend to use React, JavaScript, Elixir, and Clojure these days. We will usually be responsible for setting up a customer’s infrastructure on AWS with tools like Ansible and Terraform though sometimes we opt for a Platform as a Service like Heroku. We make sure there is a Continuous Delivery pipeline running our automated test suites before deploying changes.
+
+Reliability Engineers will split their time between responding to automated alerts ensuring the uptime of our customers applications, proactive codebase improvements, coaching and mentoring delivery teams to build production-ready systems and championing reliability engineering culture more widely.
+
+This role requires less of a physical presence with customers and therefore it is our first fully remote engineering role. You are of course welcome to work from our offices rather than remote but the option is open. As previously mentioned, regular communication with customers is still a must.
```
Comment (chrisblackburn) [inline roles/senior_reliability_engineer.md]: I'm not sure this is previously mentioned?

File: roles/senior_reliability_engineer.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+## What experience are we looking for?
+
+We’ll expect you to be a polyglot programmer versed in a mix of object and functional programming languages. We expect you to be able to jump into a project in a language, framework or configuration you’ve not worked in before and be able to fix issues, improve the codebase and build new features in it. Not only this but the right person would be adept in sharing their knowledge with others – we’d love to hear some examples of coaching and growing team members.
+
+Multitasking, or more accurately, prioritisation is a big part of being a Reliability Engineer. We expect you to be able to prioritise your work, be able to switch onto resolving emergencies without being told to, and keep all customers up to date.
+
+Code is only the half of it. The other half is communication. We expect you to be a great communicator with customers and colleagues in written and spoken English. Reliability Engineers keep in touch with customers mostly via Slack and Google Hangouts.
```
Comment (chrisblackburn) [inline roles/senior_reliability_engineer.md]: Should we be switching language to be primarily about interfacing with delivery teams over customers?

File: roles/senior_reliability_engineer.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+### To proactively identify technical improvements and recommend them to delivery teams
+
+We expect our Reliability Engineers to be able to spot areas for improvement in codebases such as test suites, dependency upgrades, performance issues detected by monitoring. We also expect them to be able to highlight these issues in a jargon free way and be able to convey the relative urgency of the issues ultimately leading to codebases being improved.
+
+### To autonomously deliver working software
+
+We expect our Reliability Engineers to be able to work on entire features, from conception through to production deployment, without guidance and direction from others. This doesn't mean they have to be delivered without pairing, or asking advice from the wider team. What it does mean is the primary force for ensuring the feature is built and deployed in a timely fashion is the responsiblity of the Reliability Engineer working on the problem.
```
Comment (chrisblackburn) [inline roles/senior_reliability_engineer.md]: Is feature the right language for Reliability Engineers? In my head RE shouldn't be about delivering features

File: roles/senior_reliability_engineer.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+### To be fully billable by timesheeting 7 hours daily
+
+We expect our Reliability Engineers to ensure they are timesheeting 7 hours daily, 4 days a week. As our Reliability Engineers rotate around teams we need to ensure we remain profitable.
```
Comment (chrisblackburn) [inline roles/senior_reliability_engineer.md]: This should be inline with delivery teams now - so 4.5 days per week

Comment (lukemorton) [reply roles/senior_reliability_engineer.md]: Will update this separately.

Comment (lukemorton) [reply roles/senior_reliability_engineer.md]: See https://github.com/madetech/handbook/issues/194

**Comment by lukemorton** [comment]
@chrisblackburn Updated. Awaiting your approval.

**Comment by lukemorton** [comment]
@yaseminercan Your approval would also be appreciated.

**Pull request merged**